release.sh

```diff
@@ -223,6 +214,24 @@ echo "You want to merge 'merge_master_into_develop_after_release_${MVN_VERSION_R
 echo "When you have created the PR you can press any key to continue. It's ok to merge the PR later, so wait for the Jenkins tests to finish."
 read -n 1 -s -r -p "Press any key to continue (ctrl+c to cancel)"; printf "\n\n";
 
+git checkout $MVN_VERSION_RELEASE
+
+echo "";
+echo "One-jar build takes several minutes, this is the last step and the script will complete by itself when you continue."
+read -n 1 -s -r -p "Press any key to continue to one-jar build (ctrl+c to cancel)"; printf "\n\n";
+
+# build one jar
+mvn -Passembly clean install -DskipTests
+
```

Comment:
This doesn't just build the onejar, it also builds the SDK zip. And the problem is that due to a glitch in our javadoc aggregate configuration, you need to run command twice: once in the project root, and then a second time _inside_ the `assembly` directory. If you skip that second step, the SDK zip still gets created, but it does not include the javadoc documentation.

Reply:
Ok. I’ll adjust that.